app/renderer/views/Explorer/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import Page from '../Page/index'
```

Comment:
No need to explicitly reference index in the import statement if index.js is present in this folder